prime-angular-frontend/src/app/shared/components/forms/site-information-form/site-information-form.component.ts

```diff
@@ -81,4 +82,8 @@ export class SiteInformationFormComponent implements OnInit {
       )
       .subscribe();
   }
+
+  private getSiteOperational(): void {
+    this.form.get('activeBeforeRegistration').valueChanges.subscribe();
```

Comment:
seems valueChanges is subscribed but did not do anything?

Reply:
removed